packages/fx-core/resource/package.nls.json

```diff
@@ -888,7 +888,7 @@
   "driver.aadApp.log.skipGenerateClientSecret": "Environment variable %s already exist, skipping Azure Active Directory app client secret generation step.",
   "driver.aadApp.log.outputAadAppManifest": "Build Azure Active Directory app manifest completed, and app manifest content is written to %s",
   "driver.aadApp.log.successUpdateAadAppManifest": "Applied manifest %s to Azure Active Directory application with object id %s",
-  "botRegistration.ProgressBar.creatingBotAadApp": "Creating bot aad app.",
+  "botRegistration.ProgressBar.creatingBotAadApp": "Creating bot aad app...",
```

Comment:
aad should be capitalized as AAD, and this also applies to all other sentences here.

Technically we also want to use full name than the acronyms:
AAD -> Azure Active Directory, unless space is limited and there are earlier texts mentioned full name.

Reply:
@sffamily thanks, I've updated the related strings, could you please review again?